docs/sql-pipe-syntax.md

```diff
@@ -0,0 +1,162 @@
+| `SET <column> = <expression>, ...`                                                        | Updates columns of the input table by replacing them with<br/>the result of evaluating the provided expressions.                                                                                                                      |
+| `DROP <column>, ...`                                                                      | Drops columns of the input table by name.                                                                                                                                                                                             |
+| `AS <alias>`                                                                              | Retains the same rows and column names of the input table<br/>but with a new table alias.                                                                                                                                             |
+| `WHERE <condition>`                                                                       | Returns the subset of input rows passing the condition.                                                                                                                                                                               |
+| `LIMIT <n> [OFFSET <m>]`                                                                  | Returns the specified number of input rows, preserving ordering<br/>(if any).                                                                                                                                                         |
+| `AGGREGATE <agg_expr> [[AS] alias], ...`                                                  | Performs full-table aggregation, returning one result row with<br/>a column for each aggregate expression.                                                                                                                            |
+| `AGGREGATE [<agg_expr> [[AS] alias], ...]`<br/>`GROUP BY <grouping_expr> [AS alias], ...` | Performs aggregation with grouping, returning one row per group.<br/>The column list includes the grouping columns first and then the<br/>aggregate columns afterwards. Aliases can be assigned directly<br/>on grouping expressions. |
+| `[LEFT \| ...] JOIN <relation>`<br/>` [ON <condition> \| USING(col, ...)]`                | Joins rows from both inputs, returning a filtered cross-product of<br/>the pipe input table and the table expression following the<br/>JOIN keyword.                                                                                  |
+| `ORDER BY <expr> [ASC \| DESC], ...`                                                      | Returns the input rows after sorting as indicated.                                                                                                                                                                                    |
```

Comment:
Here is the doc for ORDER BY: https://spark.apache.org/docs/latest/sql-ref-syntax-qry-select-orderby.html
Do we support nulls_sort_order?

Reply:
Yes, we do support it. I added a mention for this.